Makefile

```diff
@@ -142,7 +142,7 @@ test: ginkgo-set tidy-vendor
 	@go test -tags $(GO_BUILD_TAGS) ./... --race --bench=. -cover --count=1 --vet=all
 
 test-verbose: ginkgo-set tidy-vendor
-	@go test -tags $(GO_BUILD_TAGS) -covermode=atomic -coverprofile=coverage.out -v ./... --race --bench=. -cover --count=1 --vet=all
+	@go test -tags $(GO_BUILD_TAGS) -covermode=atomic -coverprofile=coverage.out -v ./pkg/... --race --bench=. -cover --count=1 --vet=all
```

Comment:
e2e test cases without build tag, hence it will be tested once covered in test scope.

Reply:
@rootfs , I suppose we should add e2e folder into general test case. For detail, you can ref https://github.com/sustainable-computing-io/kepler/blob/main/e2e/e2e_suite_test.go#L27

Reply:
or, may I know the reason why we can't run this new added case at locally?
We have checked with restapi with url path locally, https://github.com/sustainable-computing-io/kepler/blob/main/e2e/restapicheck_test.go#L24 and for this pr, we added detail checking for URL path `/metric`
hence, I want to check the metric with details locally as well.